params: {
  max_v: 3,
  max_a: 1, 
  mode: "auto" # auto or debug
}

waypoints: {

  # x: [-39.0, -55.0],
  # y: [10.0, 1.0],
  # z: [15.0, 8.0]

  # x: [-39.0, -55.0, -70.0, -150.0, -250.0, -321.0, -330.0],
  # y: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 5.0],
  # z: [15.0,20.0, 25.0, 25.0, 20.0, 15.0, 15.0]

  x: [-39.0, -55.0, -70.0, -150.0, -250.0, -321.0],
  y: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0],
  z: [15.0,20.0, 25.0, 25.0, 20.0, 15.0]

}